README.md

```diff
@@ -0,0 +1,4 @@
+@codesignal/meteor-protomongo
+=
+
+Adds asynchronous methods to `meteor/mongo`.
```

Comment:
Please add descriptions for how to install this package and use it, along with a list of the methods added to the prototype. It will be useful not only for any open-source users, but also for ourselves the next time we want to install this package in a new Meteor app

Reply:
It would also be worthwhile to document instructions for how to publish new versions of this package (in particular, how to run the build pipeline)